src/components/atoms/PageDescription.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export function PageDescription(props) {
```

Comment:
Make sure you add a comment for the component description too

Reply:
Thanks, I'll make sure to document my work in the future 👍